src/collection.js

```diff
@@ -297,11 +297,11 @@ export default class Collection {
    * @param  {Object} [options.headers] The headers object option.
    * @return {Promise<Object, Error>}
    */
-  getRecord(id, options = {}) {
+  async getRecord(id, options = {}) {
     const path = endpoint("record", this.bucket.name, this.name, id);
     const reqOptions = this._collOptions(options);
     const request = { ...reqOptions, path };
-    return this.client.execute(request);
+    return await this.client.execute(request);
```

Comment:
It seems to me like we could have missed one of these and everything would still behave the same way. Is that correct? Is there any eslint support for making sure that any `async` function has to be `await`'ed? I guess there are use cases when you don't want to, but they seem pretty rare to me.